src/Symfony/Component/Workflow/MarkingStore/MarkingStoreInterface.php

```diff
@@ -38,5 +38,5 @@ public function getMarking($subject);
      *
      * @param object $subject A subject
      */
-    public function setMarking($subject, Marking $marking, array $context = []);
+    public function setMarking($subject, Marking $marking /*, array $context = []*/);
```

Comment:
please add the argument in the docblock: this will tell `DebugClassLoader` to trigger a deprecation notice to hint ppl they should add the extra argument now.